app/representers/api/distribution_representer.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+class Api::DistributionRepresenter < Api::BaseRepresenter
+  property :id, writeable: false
+  property :type
+  property :guid
+  property :url
+  hash :properties
```

Comment:
A nested-object inside a HAL resource, huh?  I don't see anything in the spec prohibiting it, but there are some comments in the `hal-discuss` google group frowning on it.  Opinions?  Do we do this anywhere else in CMS?